package/MDAnalysis/auxiliary/XVG.py

```diff
@@ -19,343 +19,348 @@
 # MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
 # J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
 #
-

```

Comment:
I just removed the `\r` at the end of the lines. The file seems to be ASCII as it should be.